tachyon/zk/plonk/permutation/permutation_prover_impl.h

```diff
@@ -158,6 +161,9 @@ void PermutationProver<Poly, Evals>::Open(
 #define OPENING(poly, point) \
   base::Ref<const Poly>(&poly), point##_ref, poly.Evaluate(point_set.point)
 
+  // NOTE(chokobole): The ORDER IS IMPORTANT!!!
```

Comment:
```suggestion
  // The ORDER IS IMPORTANT!!!
```
I thought we were removing NOTE(author) comments. https://github.com/kroma-network/tachyon/commit/98db4aa004b6202d9ca0660b74456f30f1dc2102